source/shared/cpp/ObjectModel/Container.cpp

```diff
@@ -49,29 +37,24 @@ void Container::SetLanguage(const std::string& value)
     PropagateLanguage(value, m_items);
 }
 
-VerticalContentAlignment Container::GetVerticalContentAlignment() const
-{
-    return m_verticalContentAlignment;
-}
-
-void Container::SetVerticalContentAlignment(const VerticalContentAlignment value)
-{
-    m_verticalContentAlignment = value;
-}
-
 Json::Value Container::SerializeToJsonValue() const
 {
     Json::Value root = BaseCardElement::SerializeToJsonValue();
 
-    if (m_style != ContainerStyle::None)
+    if (GetStyle() != ContainerStyle::None)
     {
-        root[AdaptiveCardSchemaKeyToString(AdaptiveCardSchemaKey::Style)] = ContainerStyleToString(m_style);
+        root[AdaptiveCardSchemaKeyToString(AdaptiveCardSchemaKey::Style)] = ContainerStyleToString(GetStyle());
     }
 
-    if (m_verticalContentAlignment != VerticalContentAlignment::Top)
+    if (GetVerticalContentAlignment() != VerticalContentAlignment::Top)
     {
         root[AdaptiveCardSchemaKeyToString(AdaptiveCardSchemaKey::VerticalContentAlignment)] =
-            VerticalContentAlignmentToString(m_verticalContentAlignment);
+            VerticalContentAlignmentToString(GetVerticalContentAlignment());
+    }
+
+    if(GetBleed())
+    {
+        root[AdaptiveCardSchemaKeyToString(AdaptiveCardSchemaKey::Bleed)] = true;
     }
```

Comment:
Can we put this in the parent class?

Reply:
(and style and vertical content alignment too)

---
In reply to: [258662751](https://github.com/Microsoft/AdaptiveCards/pull/2406#discussion_r258662751) [](ancestors = 258662751)